scvi/inference/total_inference.py

```diff
@@ -618,13 +1052,97 @@ def loss(self, tensors):
             + self.pro_recons_weight * reconst_loss_protein
             + self.kl_weight * kl_div_z
             + kl_div_l_gene
```

Comment:
that simplifies a bit that this is the same

Reply:
Not sure what you mean?

Reply:
it's great that you're using the same parameter